src/components/poolstats/PoolPieChartWidget.tsx

```diff
@@ -33,68 +25,20 @@ const PieChartContainer = styled.div`
   transform: rotate(90deg);
 `;
 
-const useMove = () => {
-  const [state, setState] = useState<MouseMoveData>({
-    x: 0,
-    y: 0,
-    r: 0,
-    theta: 0,
-    isActive: false,
-  });
-
-  const handleMouseMove: MouseEventHandler<HTMLDivElement> = (e) => {
-    const currentTarget = e.currentTarget;
-    const boundingRect = currentTarget.getBoundingClientRect();
-
-    // Compute pointer's offset from center of element (in pixels)
-    const offsetX =
-      e.clientX - (boundingRect.x + currentTarget.offsetWidth / 2);
-    const offsetY =
-      e.clientY - (boundingRect.y + currentTarget.offsetHeight / 2);
-
-    // Non-dimensionalize it (-1 to 1)
-    const percentX = (offsetX * 2) / currentTarget.offsetWidth;
-    const percentY = (offsetY * 2) / currentTarget.offsetHeight;
-
-    const r = Math.hypot(percentX, percentY);
-    let theta = (Math.atan2(-percentX, percentY) * 180) / Math.PI;
-    if (theta < 0) theta += 360;
-
-    setState((state) => {
-      const data: MouseMoveData = {
-        ...state,
-        x: percentX,
-        y: percentY,
-        r,
-        theta,
-        isActive: true,
-      };
-
-      return data;
-    });
-  };
-
-  const handleMouseLeave: MouseEventHandler<HTMLDivElement> = (e) => {
-    setState((state) => ({ ...state, isActive: false }));
-  };
-
-  return {
-    mouseData: state,
-    handleMouseMove,
-    handleMouseLeave,
-  };
-};
-
 // MARK: Pie chart setup -------------------------------------------------------------
 
 type PieChartSlice = {
+  index: number;
```

Comment:
`index` as a property of `PieChartSlice` makes sense, as it's general purpose and makes it possible to remove the mouseMoveData atan2 stuff. Nice work